### PR TITLE
Issue #93 Fix the credential id for CRC_PULL_SECRET

### LINKED ISSUE #93: Fix the credential id for `CRC_PULL_SECRET`

### PR DESCRIPTION
**Pull request by anjannath**
Fixes #93

### CODE REVIEW

File: minishift-ci-index.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@
                 credential-id: minishift_secret_password
```
Comment (gbraad) [inline minishift-ci-index.yaml]: Minishift ? UGH

**Comment by anjannath** [comment]
retest this please

File: minishift-ci-index.yaml (hunk 1 above)
Comment (anjannath) [reply minishift-ci-index.yaml]: This was added by the maintainer, we'll remove these from here any ways once we get the CRC namespace in centos ci

File: minishift-ci-index.yaml [hunk 2]
```diff
@@ -26,7 +26,7 @@
...
                 variable: CRC_BUNDLE_PASSWORD
             - text:
-                credential-id: crc_pull_secret
+                credential-id: code_ready_container_pull_secret
```
Comment (praveenkumar) [inline minishift-ci-index.yaml]: :+1:

**Pull request merged**